opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -385,12 +384,25 @@ where
                 .clone()
                 .map(|allowed| Arc::new(move |kv: &KeyValue| allowed.contains(&kv.key)) as Arc<_>);
 
-            let b = AggregateBuilder::new(self.pipeline.reader.temporality(kind), filter);
+            let cardinality_limit = stream
+                .cardinality_limit
+                .unwrap_or(DEFAULT_CARDINALITY_LIMIT);
+            let b = AggregateBuilder::new(
+                self.pipeline.reader.temporality(kind),
+                filter,
+                cardinality_limit,
+            );
             let AggregateFns { measure, collect } = match aggregate_fn(b, &agg, kind) {
                 Ok(Some(inst)) => inst,
                 other => return other.map(|fs| fs.map(|inst| inst.measure)), // Drop aggregator or error
             };
 
+            otel_debug!(
+                name : "Metrics.InstrumentCreated",
+                instrument_name = stream.name.as_ref(),
```

Comment:
nit: Add `Aggregation` information as well to understand what kind of instrument was created.